network/src/topology.rs

```diff
@@ -86,11 +86,20 @@ impl Connection {
     }
 }
 
+/// Message types passed through the `KnownNetwork` channel.
+pub enum KnownNetworkMessage {
+    Peers((SocketAddr, Vec<SocketAddr>)),
```

Comment:
very much a nit, buuut
```suggestion
    Peers(SocketAddr, Vec<SocketAddr>),
```
and the same for `KnownNetworkMessage::Height`

Reply:
the enum's  could also use a short description regarding their components